mysql-compatibility.md

```diff
@@ -62,7 +62,7 @@ However, some features of MySQL are not supported. This could be because there i
 
 ### Auto-increment ID
 
-+ In TiDB, auto-incremental columns are globally unique. They are incremental on a single TiDB server, but *not* necessarily incremental among multiple TiDB servers or allocated sequentially. It is recommended that you do not mix default values and custom values. Otherwise, you might encounter the `Duplicated Error` error message.
++ In TiDB, auto-incremental columns are globally unique. They are incremental on a single TiDB server. If you want the numbers to be incremental among multiple TiDB servers, you can use the [`AUTO_INCREMENT` MySQL compatibility mode](/auto-increment.md#mysql-compatibility-mode). But it is not necessarily allocated sequentially. It is recommended that you do not mix default values and custom values. Otherwise, you might encounter the `Duplicated Error` error message.
```

Comment:
```suggestion
+ In TiDB, auto-incremental columns are globally unique. They are incremental on a single TiDB server. If you want the numbers to be incremental among multiple TiDB servers, you can use the [`AUTO_INCREMENT` MySQL compatibility mode](/auto-increment.md#mysql-compatibility-mode). But the numbers are not necessarily allocated sequentially. It is recommended that you do not mix default values and custom values. Otherwise, you might encounter the `Duplicated Error` error message.
```

Reply:
ID vs number. Which one is more accurate? According to the title, we should use ID...

Reply:
@shichun-0415 I have modified the `numbers` to `IDs`. PTAL again

Reply:
LGTM